app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1 class="text-center py-4">Recipes list</h1>
+
+<%= link_to "Add New Recipe", new_recipe_path, class: "btn btn-primary mb-2 float-end" %>
+<% @recipes.each do |recipe| %>
+  <div class="card border-secondary mb-2" style="width: 100%;">
+    <div class="card-body text-secondary d-flex justify-content-between align-items-center">
+      <div style="width: 30%;">
+        <h5 class="card-title">
+          <a href="<%= recipe_path(recipe) %>" class="text-decoration-none text-uppercase font-weight-bold">
+            <%= recipe.name %>
+          </a>
+        </h5>
+        <%= button_to "REMOVE", recipe_path(recipe), method: :delete, data: { turbo_confirm: "Are you sure? "}, class: "text-decoration-none btn btn-danger" %>
+      </div>
+
+      <p style="width: 70%;"><%= recipe.description %></p>
+
+      <!-- Calculate and display total amount for the recipe -->
+      <p>Total Amount: $<%= recipe_total_amount(recipe) %></p>
+    </div>
+  </div>
```

Comment:
- [ ] As pointed out by the last reviewer, the error of not having the right number added persists.
- Secondly, from the figure below, I added `15` and `10`, but there seems to be added numbers to it as a bug.Kindly fix this 👍 

### Error view:
![image](https://github.com/GhausAliKhan/recipe-app/assets/50990925/fceb3596-d5ec-4d34-ac80-877981da9d82)

Reply:
Hi dear @Shedrack-Sunday 
I hope you doing well!
As you see in the picture below, I added 2, 12, and 24 numbers and I has the correct number now.
What do you think?
![new](https://github.com/GhausAliKhan/recipe-app/assets/119149574/0ffcebfb-d18a-4f27-9ba5-6c915ef5a457)

Reply:
You don't have to necessarily input the piece together with the quantity. Please follow the wireframe and input just one figure 🙏